### DISCUSSION TITLE
General question regarding internet / cloud server connection

### DISCUSSION THREAD

**Discussion by helgek** [Michsior14/ha-venta#40] [Q&A] [answered]
For my understanding since I have not yet tried your integration with my newly received AH510 (and it's currently lying next to me in disassembled pieces) I wanted to ask if the protocol communication also works if the Venta appliance can't connect / isn't connected to the cloud server? I assume yes but just wanted to confirm. Maybe it would be also good to reflect this in readme. Thanks!

Additional note: I just found this information: https://github.com/Michsior14/ha-venta/issues/1#issuecomment-1858937681

Do I get this right that the wifi settings can be adjusted completely without signing up through the app? This would then probably mean that the device is already cloud free as it is. Nevertheless my ESPHome proposal in the other post may still be of interest. Have you maybe become aware already of an OTA interface that could be used to flash the ESP32 without accessing the PCB?

**Comment by Michsior14** [comment, marked as answer]
Hey, yes as far as I know all Venta devices are communicating locally only, but on some devices you have to setup wifi through the app (on some you can connect to the device as [described here](https://github.com/GreyEarl/venta-air-api?tab=readme-ov-file#connecting-to-your-network)). I've personally blocked internet access completely for the device and there are no issues whatsoever ;)

Regarding ESP32, I did not dig into this at all.

**Reply by Michsior14** [reply]
Ah regarding the readme, I am open to any contributions! Since you already created the PR, you can add some details there you think are missing.

**Reply by helgek** [reply]
Thank you! Indeed with my not yet set up device I was able to connect to it's AP and then connect through the web interface to my WiFi. So I didn't need the app at all.

<img width="831" alt="image" src="https://github.com/Michsior14/ha-venta/assets/6280991/57650fce-3a7c-4455-bf33-a6e5e4c33d7e">

an IP was asigned to my computer by the Venta AP it seems:

<img width="444" alt="image" src="https://github.com/Michsior14/ha-venta/assets/6280991/e5ddfd0c-e80d-41c7-a575-33f5a0fb76e6">
